pkgs/by-name/bu/buildstream/package.nix

```diff
@@ -34,26 +34,28 @@ python3Packages.buildPythonApplication rec {
     setuptools-scm
   ];
 
-  dependencies = with python3Packages; [
-    click
-    dulwich
-    grpcio
-    jinja2
-    markupsafe
-    packaging
-    pluginbase
-    protobuf
-    psutil
-    pyroaring
-    requests
-    ruamel-yaml
-    ruamel-yaml-clib
-    tomlkit
-    ujson
-  ];
+  dependencies =
+    [ buildbox ]
+    ++ (with python3Packages; [
+      click
+      dulwich
+      grpcio
+      jinja2
+      markupsafe
+      packaging
+      pkgs.buildbox
+      pluginbase
```

Comment:
```suggestion
  dependencies =
    [ buildbox ]
    ++ (with python3Packages; [
      click
      dulwich
      grpcio
      jinja2
      markupsafe
      packaging
      pluginbase
```

redundant

Reply:
Oh, oops. Thought I removed that! I'll edit.